file-formats/doma/type/zif_aff_doma_v1.abap

```diff
@@ -0,0 +1,115 @@
+INTERFACE zif_aff_doma_v1 PUBLIC.
+
+  "! <p class="shorttext">Plus - Minus sign for domains</p>
+  "! Plus - Minus sign for domains
+  TYPES ty_negative_values TYPE abap_bool.
+
+  "! <p class="shorttext">Case sensitive flag for domains</p>
+  "! Case sensiive flag for domains
+  TYPES ty_case_sensitive       TYPE abap_bool.
+
+  "! <p class="shorttext">Data type for the domain</p>
+  "! Data type for the domain
+  TYPES ty_data_type            TYPE c LENGTH 4.
+
+  "! <p class="shorttext">No Characters</p>
+  "! Number of Characters
+ "! $minimum 0
+ "! $maximum 999999
+ TYPES ty_numb_char            TYPE i.
```

Comment:
Since we changed the field `numb_char` to `length`, I suggest to also rename the type to `ty_length`

```suggestion
 TYPES ty_numb_char            TYPE i.
```

Reply:
I'm sorry I'm kind of lost, you mean

```suggestion
 TYPES ty_numb_char            TYPE ty_length.
```
?

Reply:
Sorry, my fault. I didn't adapt my suggestion. This is what I meant 
```suggestion
 TYPES ty_length            TYPE i.
```

Reason: We renamed the type for field `length` to `ty_length` but didn't adapt the type.

P.S.: I remove my previous suggestion, because it does not make any sense :(

Reply:
done! :)